sources/th.json

```diff
@@ -12,5 +12,19 @@
             "info@openaq.org"
         ],
         "active": true
+    },
+    {
+        "url": "http://air4thai.pcd.go.th/forappV2/getAQI_JSON.php",
+        "adapter": "air4thai",
+        "name": "Air4Thai",
+        "city": "",
+        "country": "TH",
+        "description": "",
```

Comment:
I would add the agency name in description so it's there "Air Quality and Noise Management Bureau, Pollution Control Board"